src/main/java/seedu/address/model/transaction/Transaction.java

```diff
@@ -42,4 +42,9 @@ public Date getDate() {
         return date;
     }
 
+    @Override
+    public String toString() {
+        return goods.toString();
```

Comment:
Perhaps you could write a test for this line.

Reply:
Alternatively, you could remove the `Transaction#toString()` since there is no need for it.

Reply:
You're right, I've just removed the toString() method as it is not required. Thanks